### PR TITLE
fix(learn): add note about tweet button restrictions

### LINKED ISSUE #40089: Quote Machine Tweet button doesn't work on CodePen due to Twitter's API
**Describe your problem and how to reproduce it:**
The Tweet button on the Quote Machine project isn't working due to twitter.com refusing to connect and a ERR_BLOCKED_BY_RESPONSE error.
User story 10 in the instructions for this project states

> I can tweet the current quote by clicking on the `#tweet-quote` `a` element. This `a` element should include the `"twitter.com/intent/tweet"` path in its `href` attribute to tweet the current quote.

So I did some digging in Twitter’s web intents documentation and found that **web intents cannot be loaded inside an iframe** (about halfway through Working with Web Intents section).

I looked into how CodePen renders its projects, and in both the editor and full views, it renders as an iframe. Since the user has no control over Twitter’s rules, or how CodePen renders the project, I don’t know how a tweet button could be successfully implemented in this project. Maybe posting to another social media site could be implemented instead?

**Add a Link to the page with the problem:**
- [FreeCodeCamp quote machine instructions](https://www.freecodecamp.org/learn/front-end-libraries/front-end-libraries-projects/build-a-random-quote-machine)
- [FreeCodeCamp official example](https://codepen.io/freeCodeCamp/full/qRZeGZ)
- [Twitter documentation for web intents](https://developer.twitter.com/en/docs/twitter-for-websites/web-intents/overview)

**Tell us about your browser and operating system:**
* Browser Name: Google Chrome
* Browser Version: 86.0.4240.111
* Operating System: MacOS Catalina (10.15.6)

**If possible, add a screenshot here (you can drag and drop, png, jpg, gif, etc. in this box):**

Screen Shot 2020-10-25 at 4 24 41 PM

### PR DESCRIPTION
**Pull request by IamSurrett**
Checklist:

- [x] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `master` branch of freeCodeCamp.
- [x] All the files I changed are in the same world language, for example: only English changes, or only Chinese changes, etc.

Closes #40089 

I added the paragraph that starts with "**Note**" in the picture below. This screenshot was taken from my local freeCodeCamp client.
Screen Shot of quote machine instructions

### CODE REVIEW

File: curriculum/challenges/english/03-front-end-libraries/front-end-libraries-projects/build-a-random-quote-machine.md [hunk 1]
```diff
@@ -22,6 +22,7 @@ You can use any mix of HTML, JavaScript, CSS, Bootstrap, SASS, React, Redux, and
 <strong>User Story #10:</strong> I can tweet the current quote by clicking on the <code>#tweet-quote</code><code>a</code> element. This <code>a</code> element should include the <code>"twitter.com/intent/tweet"</code> path in its <code>href</code> attribute to tweet the current quote.
 <strong>User Story #11:</strong> The <code>#quote-box</code> wrapper element should be horizontally centered. Please run tests with browser's zoom level at 100% and page maximized.
 You can build your project by forking <a href='https://codepen.io/freeCodeCamp/pen/MJjpwO' target='_blank'>this CodePen pen</a>. Or you can use this CDN link to run the tests in any environment you like: <code>https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js</code>
+<strong>Note:</strong> If you are developing this project on CodePen, restrictions prevent the Twitter link from loading in the same window. Use the <code>target="_blank"</code> attribute on the <code>#tweet-quote</code> element as a workaround.
```
Comment (RandellDawson) [inline curriculum/challenges/english/03-front-end-libraries/front-end-libraries-projects/build-a-random-quote-machine.md:25]: The note should be at the bottom of the `description` section.

**Comment by ieahleen** [comment]
the only issue I have with this is that it doesn't actually work. Even with `target="_blank"` there is the same error from twitter

**Comment by moT01** [comment]
What browser are you using @ieahleen? Wondering if we should make the note more generic. Something like: ``Note: Twitter does not allow links to be loaded in an iframe. Try using `target="_blank"` as a workaround.``

**Comment by ieahleen** [comment]
@moT01 this was the discussion on the forum: https://forum.freecodecamp.org/t/codepen-adding-domain-to-my-links/427877

my browser is Google Chrome Versione 86.0.4240.111 (Build ufficiale) (a 64 bit)

**Comment by moT01** [comment]
I vote to go with @ieahleen's suggestion on the issue (actually, I modified it slightly - I'm fine her suggestion or anything along these lines):
``Note: Twitter does not allow links to be loaded in an iframe. Try using the `target="_blank"` or `target="_top"` attribute on the `#tweet-quote` element if your tweet won't load. `target="_top"` will replace the current tab so make sure your work is saved.``

I suppose we could probably add `target="top"` to the example project as well.

**Review by moT01 — DISMISSED** [review]
We should modify this to use something along the lines of what @ieahleen suggested

**Comment by IamSurrett** [comment]
I modified the hint to @moT01's suggestion and put it at the bottom of the `description` section 

Screen Shot 2020-11-08 at 6 31 52 PM

File: curriculum/challenges/english/03-front-end-libraries/front-end-libraries-projects/build-a-random-quote-machine.md (hunk 1 above)
Comment (IamSurrett) [reply curriculum/challenges/english/03-front-end-libraries/front-end-libraries-projects/build-a-random-quote-machine.md]: 👍🏻

**Review by naomi-lgbt — APPROVED** [review]
LGTM~! 🥳 I think this clarifies the issue well.

Congratulations on your first pull request to this project! We look forward to future contributions!

Please note that our workflow requires at least one additional approving review before this can be merged.

**Review by majestic-owl448 — APPROVED** [review]
I checked the changes, and as there is already an approving review I am also going to merge. Congratulations for your accepted contribution!

**Comment by ieahleen** [comment]
I can't merge because of @moT01 requested change 😅

**Comment by Sky020** [comment]
Hi @IamSurrett . Congrats on your first pull request (PR)! 🎉

Thank you, for your contribution to the page! 👍
We are happy to accept these changes and look forward to future contributions. 📝

**Pull request merged**